### DISCUSSION TITLE
Add this to f-droid or izzyondroid repo

### DISCUSSION THREAD

**Discussion by Adithyan007hack** [robertodoering/harpy#765] [Ideas]
This a fantastic app that respect privacy.
But it is only available through playstore , a privacy concern person will think twice before installing anything from google ecosystem.
It would be convinient of you add this to f-roid or it is a beta version so for faster updates we can add it in izzyondroid also.
Hope you understand (if you are willing we can ping izzyondroid to notice this app to add it in his repo)

**Comment by robertodoering** [comment]
There are no plans to bring harpy to f-droid at the moment but if you want to avoid the play store you can download the latest apk here:
https://github.com/robertodoering/harpy/releases
